charts/rancher-vsphere-csi/rancher-vsphere-csi/2.3.1/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+annotations:
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: vSphere CSI
+  catalog.cattle.io/namespace: kube-system
+  catalog.cattle.io/os: linux
+  catalog.cattle.io/rancher-version: '>= 2.5.0-0 <= 2.5.99-0'
+  catalog.cattle.io/release-name: vsphere-csi
+apiVersion: v1
+appVersion: 2.3.0
```

Comment:
This should be 2.3.1 to match `version`

Reply:
@PennyScissors @aiyengar2  Should update the value to 2.3.1?

Reply:
Let's keep it since we've always had it. Let's make it 2.3.1